meeting-notes/09-14February2019.md

```diff
@@ -0,0 +1,151 @@
+  those. He has also interest on ipfs, ipld and resilent networks.
+- Chris says hi. He also works on Wireline. He has lot of experience
+  working with communities and figuring out how to help the decentralized
+  ecosystem grow.
+- Georgy introduces himself. He is an independent contractor, currently
+  working for the Dat Project. He is helping with docs, dat-js
+  improvements and more.
+- Martin is one of the [GEUT](https://github.com/geut) co-founders. He is
+  also currently working for Wireline exploring new ideas and PoCs.
+- Gina says hi. She is working at [Rumours](https://rumo.rs/), he also
+  collaborates on CS&S and is one of the organizers of Dat nights.
+- Roy introduces himself. He is working for Diglife. They have many
+  interests, one of these is exploring "tech of trust" niche. He is also
+  digging into Dat these days.
+- Diego mentions the [new blog
+  post](https://blog.datproject.org/2019/02/13/handshake-funding-brings-in-new-developers-for-dat/)
+  from Dat Project about the recent grants. He asks Georgy, one of the
+  grant holders, if he can tell us more about his new tasks on the Dat
+  Project.
+  - Georgy told us about [dat-js](https://github.com/datproject/dat-js)
+    which is what he is working on these days. Making some improvements
+    around. They need those fixes for the [internet archive](https://archive.org/), they plan to
+    start using Dat on the web pretty soon. He also plans to start working
+    on a new mobile app targeted for Android users similar to Beaker
+    browser.
+- A bit of identity on Dat
+  - Roy asks Georgy about identity and how he plans to solve it.
+  - Georgy mentions that his current work is not concerned about identity.
+    He says that is a problem that can be solved at a different level, like
+    application level. He also mention that Paul Frazee (Beaker) has been
+    thinking about identity and it is a better candidate for that kind of
+    questions.
+  - Rich mentions that they are working trying to solve decentralized
+    identity on Wireline. They have a credentials mechanism. They are also
+    looking for some wallet implementations. Wireline is about to publish
+    a new paper soon.
+- libp2p & Dat
+  - Rich mentions that they are working trying to solve decentralized
+    identity on Wireline. They have a credentials mechanism. They are also
+    looking for some wallet implementations. Wireline is about to publish
+    a new paper soon.
+  - Rich also mentions libp2p, the whole stack and ipld, how it is suitable
+    for semantic web and deep linking. He is curious about the relation of
+    these technologies and Dat.
+  - Georgy mentions that in the past, he has some interest on adding
+    libp2p support to Dat. But that idea never gain traction in the Dat
+    community. He also doesn't think that currently Dat could go in that
+    direction. In fact, new modules like the whole [hyperswarm](https://github.com/hyperswarm/) suggest the
+    opposite.
```

Comment:
@RangerMauve I looked into libp2p and am not too happy about some of its features. Aside from the fact that there is no "unannounce" which is kinda vital to DAT: It also comes with a huge baggage of code that is nice to have, but adds a whole lot of complexity for little gain. I imagine the start-up or maintenance costs are not that trivial. IPFS seems to have the money to afford that though.

Reply:
Yeah, I agree. I really like how they're thinking far into the future and have a strong focus on multiple transports and encoding schemes, but it does end up having a lot of moving parts and  the highly dynamic nature is less focused.